devimint/src/devfed.rs

```diff
@@ -354,4 +284,18 @@ impl DevJitFed {
         );
         Ok(())
     }
+
+    pub async fn to_dev_fed(self, process_mgr: &ProcessManager) -> anyhow::Result<DevFed> {
+        self.finalize(process_mgr).await?;
+        Ok(DevFed {
+            bitcoind: self.bitcoind().await?.to_owned(),
+            cln: self.cln().await?.to_owned(),
+            lnd: self.lnd().await?.to_owned(),
+            fed: self.fed().await?.to_owned(),
+            gw_cln: self.gw_cln().await?.to_owned(),
+            gw_lnd: self.gw_lnd().await?.to_owned(),
+            esplora: self.esplora().await?.to_owned(),
+            electrs: self.electrs().await?.to_owned(),
+        })
+    }
```

Comment:
The beauty of `Jit`-types: not needing `join!` here (I was about to complain the lack thereof, but then remembered :rofl: )